apio-consumer/src/main/kotlin/com/liferay/apio/consumer/util/RequestUtil.kt

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.apio.consumer.util
+
+import com.google.gson.Gson
+import com.liferay.apio.consumer.authenticator.ApioAuthenticator
+import com.liferay.apio.consumer.exception.ApioException
+import com.liferay.apio.consumer.exception.ThingNotFoundException
+import com.liferay.apio.consumer.model.Thing
+import com.liferay.apio.consumer.model.get
+import okhttp3.*
+import java.io.ByteArrayOutputStream
+import java.io.InputStream
+import java.net.URLConnection
+
+/**
+ * @author Javier Gamarra
+ * @author Paulo Cruz
+ */
+internal class RequestUtil {
+
+    companion object {
+        fun createRequest(httpUrl: HttpUrl?, authenticator: ApioAuthenticator?): Request {
+            val request = Request.Builder()
+                .url(httpUrl!!)
+                .addHeader("Accept", "application/ld+json")
+                .build()
+
+            return authenticator?.authenticate(request) ?: request
+        }
+
+        fun createRequest(httpUrl: HttpUrl?, method: String, requestBody: RequestBody?,
```

Comment:
Why do you think about creating only one method with default values instead of 2?

Reply:
I did it, it's on the new commits. Thanks for the suggestion :)